linq4j/src/main/java/org/apache/calcite/linq4j/tree/ExpressionType.java

```diff
@@ -146,12 +146,30 @@ public enum ExpressionType {
    */
   DivideChecked(" / ", false, 3, false),
 
+  /**
```

Comment:
Why do we need all these new entries here. I didn't see any conditionals based on those?